examples/volumes/vsphere/vsphere-volume-sc-vsancapabilities.yaml

```diff
@@ -0,0 +1,9 @@
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: fast
+provisioner: kubernetes.io/vsphere-volume
+parameters:
+    diskformat: zeroedthick
+    hostFailuresToTolerate: "2"
```

Comment:
If we have two vSan Cluster as below.
- **vSan-DS1** - Host1, Host2, Host3, Host4 - K8S is deployed on this Cluster. All nodes can access vSan-DS1
- **vSan-DS2** - Host5, Host6, Host7, Host8 - Nodes can not access vSan-DS2

We should pick datastore accessible to node and also comply with the policy.

What if user specifies **vSan-DS2** in the StorageClass YAML and Default Datastore was set to **vSAN-DS1**?